week1_session4_intro_to_ml_and_scikit_learn.ipynb

```diff
@@ -0,0 +1,1547 @@
+{
```

Comment:
<p>In order to follow the scheme we talked about during the last meeting, i.e.</p><p> </p><p>1. Aims of this notebook</p><p>2. Learning goals (Theory and Practical)</p><p>3. References</p><p>4. Theory</p><p>5. Practical</p><p>6. Discussion</p><p>7. Exercises.</p><p>8. Solutions</p><p> </p><p>you could split this cell, to fill the categories:</p><p> </p><p>1. Aims of this notebook</p><blockquote>We want to introduce you to the basic concepts of machine learning and how to use python and scikit-learn to apply these concepts.</blockquote><p>Maybe mention here already the dataset you will be using.</p><p> </p><p>2. Learning goals</p><blockquote>By the end of this session, you should be familiar with:</blockquote><blockquote>the difference between supervised and unsupervised learning:</blockquote><blockquote class="ql-indent-1">classification vs regression in supervised learning</blockquote><blockquote>loading, exploring and preprocessing a dataset; specifically a csv file</blockquote><blockquote>how and why we split data into training and test sets</blockquote><blockquote>training a model on the training set</blockquote><blockquote>evaluating the performance of a model on the test set</blockquote><blockquote>using various performance measures to validate a model</blockquote><p> </p><p>3. References</p><p>TBA</p><p> </p><p>4. Theory</p><p>The theory will be covered separately with slides (link).</p><p>Maybe add a TOC</p><blockquote><strong>note:</strong> see slides for an overview of the first topic.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/ai_in_medicine/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>have made these changes :)</p>

Reply:
<p>i will most likely leave the solutions out since the steps are laid out for them above</p>